### PR TITLE
Remove country information from context validator logs

### PR DESCRIPTION
**Pull request by ioangut**
## Checklist

* [x] Test your changes as thoroughly as possible before you commit them. Preferably, automate your test by unit/integration tests.
* [x] Make sure `mvn install` runs for the whole project and, if you touched any code in the respective service, submission and distribution service can be run with `spring-boot:run`

## Description

The PR contains a cherry pick fix of the submission security vulnerability from the release/1.5.

### CODE REVIEW

**Review by KevponSAP — DISMISSED** [review]

**Review by sorin-stefan-iovita-sap — DISMISSED** [review]

File: services/submission/src/main/java/app/coronawarn/server/services/submission/validation/ValidSubmissionPayload.java [hunk 1]
```diff
@@ -127,8 +127,8 @@ private boolean checkOriginCountryIsValid(SubmissionPayload submissionPayload,
       String originCountry = submissionPayload.getOrigin();
       if (submissionPayload.hasOrigin() && !originCountry.isEmpty()
           && !supportedCountries.contains(originCountry)) {
-        addViolation(validatorContext, String.format(
-            "Origin country %s is not part of the supported countries list", originCountry));
+        addViolation(validatorContext,
+            "Key contains origin country is not part of the supported countries list");
```
Comment (EugenM-SAP) [inline services/submission/src/main/java/app/coronawarn/server/services/submission/validation/ValidSubmissionPayload.java:131]: Typo : Key contains origin country **which** is not ....

**Review by EugenM-SAP — APPROVED** [review]

**Review by KevponSAP — APPROVED** [review]

**Comment by hilmarf** [comment]
https://github.com/corona-warn-app/cwa-security/issues/63

**Comment by rugk** [comment]
The link to https://github.com/corona-warn-app/cwa-security/issues/63 is dead… :thinking:

**Comment by svengabr** [comment]
Due to NDA reasons, critical security issues are discussed internally first and will not be exposed to the public. For this reason, we have a private repository where only a small amount of people have access to.

You can read an update on our official website here:
https://www.coronawarn.app/en/blog/2020-11-19-security-update/


Thank you very much for your understanding.

Best regards,
SG

Corona-Warn-App Open Source Team

**Pull request merged**